README.md

```diff
@@ -58,13 +58,20 @@ const { getDefaultConfig } = require("expo/metro-config");
 
 module.exports = (async () => {
   const {
-    resolver: { sourceExts, assetExts }
+    transformer: { babelTransformerPath, ...transformer },
+    resolver: { sourceExts, assetExts, ...resolver },
+    ...config
   } = await getDefaultConfig(__dirname);
   return {
+    ...config,
     transformer: {
-      babelTransformerPath: require.resolve("react-native-svg-transformer")
+      ...transformer,
+      babelTransformerPath: require
+        .resolve('react-native-svg-transformer')
+        .makeTransformer(babelTransformerPath),
```

Comment:
It's not correct.
```
require.resolve(...).makeTransformer is not a function
TypeError: require.resolve(...).makeTransformer is not a function
```

Reply:
Did you apply the changes to index.js? https://github.com/kristerkari/react-native-svg-transformer/pull/151/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346

Reply:
require.resolve() return a string, so you can't call the export functions. Yes, I have patch the module and it didn't work.

Finally, I end up with custom metro-svg-transformer.js with content from the orignal transformer.